openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -253,7 +255,10 @@ const DatabaseSchemaPage: FunctionComponent = () => {
     (activeKey: string) => {
       if (activeKey !== activeTab) {
         history.push({
-          pathname: getDatabaseSchemaDetailsPath(databaseSchemaFQN, activeKey),
+          pathname: getDatabaseSchemaDetailsPath(
```

Comment:
Same here in `getDatabaseSchemaDetailsPath` we are encoding it and since `databaseSchemaFQN ` is already in an encoded format, it repeat the encoding and making the wrong URL.